[2017-02-15 12:16:35] php.DEBUG: Warning: fsockopen(): unable to connect to 127.0.0.1:8000 (Connection refused) {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/brian/projects/tht/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Command/ServerCommand.php\",\"line\":61})"} []
[2017-02-15 12:16:47] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2017-02-15 12:16:47] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-02-15 12:16:47] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"ab1de1","_route":"_wdt"},"request_uri":"http://localhost:8000/_wdt/ab1de1","method":"GET"} []
[2017-02-15 12:35:23] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Config\Exception\FileLoaderLoadException: "The routing file "/home/brian/projects/tht/app/config/routing.yml" contains unsupported keys for "fos_user": "resources". Expected one of: "resource", "type", "prefix", "path", "host", "schemes", "methods", "defaults", "requirements", "options", "condition" in /home/brian/projects/tht/app/config/routing.yml (which is being imported from "/home/brian/projects/tht/app/config/routing_dev.yml")." at /home/brian/projects/tht/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php line 118 {"exception":"[object] (Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The routing file \"/home/brian/projects/tht/app/config/routing.yml\" contains unsupported keys for \"fos_user\": \"resources\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\" in /home/brian/projects/tht/app/config/routing.yml (which is being imported from \"/home/brian/projects/tht/app/config/routing_dev.yml\"). at /home/brian/projects/tht/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:118, InvalidArgumentException(code: 0): The routing file \"/home/brian/projects/tht/app/config/routing.yml\" contains unsupported keys for \"fos_user\": \"resources\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\". at /home/brian/projects/tht/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:183)"} []
[2017-02-15 12:35:23] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\ConnectionException: An exception occured in driver: SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php line 85) {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php:85, Doctrine\\DBAL\\Driver\\PDOException(code: 7): SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 7): SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)"} []
[2017-02-15 12:35:23] php.CRITICAL: Uncaught Exception: An exception occured in driver: SQLSTATE[08006] [7] fe_sendauth: no password supplied {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php:85, Doctrine\\DBAL\\Driver\\PDOException(code: 7): SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 7): SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43, Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The routing file \"/home/brian/projects/tht/app/config/routing.yml\" contains unsupported keys for \"fos_user\": \"resources\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\" in /home/brian/projects/tht/app/config/routing.yml (which is being imported from \"/home/brian/projects/tht/app/config/routing_dev.yml\"). at /home/brian/projects/tht/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:118, InvalidArgumentException(code: 0): The routing file \"/home/brian/projects/tht/app/config/routing.yml\" contains unsupported keys for \"fos_user\": \"resources\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\". at /home/brian/projects/tht/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:183)"} []
[2017-02-15 12:35:23] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\ConnectionException: "An exception occured in driver: SQLSTATE[08006] [7] fe_sendauth: no password supplied" at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php line 85 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php:85, Doctrine\\DBAL\\Driver\\PDOException(code: 7): SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 7): SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43, Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The routing file \"/home/brian/projects/tht/app/config/routing.yml\" contains unsupported keys for \"fos_user\": \"resources\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\" in /home/brian/projects/tht/app/config/routing.yml (which is being imported from \"/home/brian/projects/tht/app/config/routing_dev.yml\"). at /home/brian/projects/tht/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:118, InvalidArgumentException(code: 0): The routing file \"/home/brian/projects/tht/app/config/routing.yml\" contains unsupported keys for \"fos_user\": \"resources\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\". at /home/brian/projects/tht/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:183)"} []
[2017-02-15 12:35:23] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\ConnectionException: An exception occured in driver: SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php line 85) {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php:85, Doctrine\\DBAL\\Driver\\PDOException(code: 7): SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 7): SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)"} []
[2017-02-15 12:35:24] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Config\Exception\FileLoaderLoadException: "The routing file "/home/brian/projects/tht/app/config/routing.yml" contains unsupported keys for "fos_user": "resources". Expected one of: "resource", "type", "prefix", "path", "host", "schemes", "methods", "defaults", "requirements", "options", "condition" in /home/brian/projects/tht/app/config/routing.yml (which is being imported from "/home/brian/projects/tht/app/config/routing_dev.yml")." at /home/brian/projects/tht/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php line 118 {"exception":"[object] (Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The routing file \"/home/brian/projects/tht/app/config/routing.yml\" contains unsupported keys for \"fos_user\": \"resources\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\" in /home/brian/projects/tht/app/config/routing.yml (which is being imported from \"/home/brian/projects/tht/app/config/routing_dev.yml\"). at /home/brian/projects/tht/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:118, InvalidArgumentException(code: 0): The routing file \"/home/brian/projects/tht/app/config/routing.yml\" contains unsupported keys for \"fos_user\": \"resources\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\". at /home/brian/projects/tht/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:183)"} []
[2017-02-15 12:35:24] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\ConnectionException: An exception occured in driver: SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php line 85) {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php:85, Doctrine\\DBAL\\Driver\\PDOException(code: 7): SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 7): SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)"} []
[2017-02-15 12:35:24] php.CRITICAL: Uncaught Exception: An exception occured in driver: SQLSTATE[08006] [7] fe_sendauth: no password supplied {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php:85, Doctrine\\DBAL\\Driver\\PDOException(code: 7): SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 7): SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43, Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The routing file \"/home/brian/projects/tht/app/config/routing.yml\" contains unsupported keys for \"fos_user\": \"resources\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\" in /home/brian/projects/tht/app/config/routing.yml (which is being imported from \"/home/brian/projects/tht/app/config/routing_dev.yml\"). at /home/brian/projects/tht/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:118, InvalidArgumentException(code: 0): The routing file \"/home/brian/projects/tht/app/config/routing.yml\" contains unsupported keys for \"fos_user\": \"resources\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\". at /home/brian/projects/tht/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:183)"} []
[2017-02-15 12:35:24] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\ConnectionException: "An exception occured in driver: SQLSTATE[08006] [7] fe_sendauth: no password supplied" at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php line 85 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php:85, Doctrine\\DBAL\\Driver\\PDOException(code: 7): SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 7): SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43, Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The routing file \"/home/brian/projects/tht/app/config/routing.yml\" contains unsupported keys for \"fos_user\": \"resources\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\" in /home/brian/projects/tht/app/config/routing.yml (which is being imported from \"/home/brian/projects/tht/app/config/routing_dev.yml\"). at /home/brian/projects/tht/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:118, InvalidArgumentException(code: 0): The routing file \"/home/brian/projects/tht/app/config/routing.yml\" contains unsupported keys for \"fos_user\": \"resources\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\". at /home/brian/projects/tht/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:183)"} []
[2017-02-15 12:35:24] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\ConnectionException: An exception occured in driver: SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php line 85) {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php:85, Doctrine\\DBAL\\Driver\\PDOException(code: 7): SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 7): SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)"} []
[2017-02-15 12:35:25] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Config\Exception\FileLoaderLoadException: "The routing file "/home/brian/projects/tht/app/config/routing.yml" contains unsupported keys for "fos_user": "resources". Expected one of: "resource", "type", "prefix", "path", "host", "schemes", "methods", "defaults", "requirements", "options", "condition" in /home/brian/projects/tht/app/config/routing.yml (which is being imported from "/home/brian/projects/tht/app/config/routing_dev.yml")." at /home/brian/projects/tht/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php line 118 {"exception":"[object] (Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The routing file \"/home/brian/projects/tht/app/config/routing.yml\" contains unsupported keys for \"fos_user\": \"resources\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\" in /home/brian/projects/tht/app/config/routing.yml (which is being imported from \"/home/brian/projects/tht/app/config/routing_dev.yml\"). at /home/brian/projects/tht/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:118, InvalidArgumentException(code: 0): The routing file \"/home/brian/projects/tht/app/config/routing.yml\" contains unsupported keys for \"fos_user\": \"resources\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\". at /home/brian/projects/tht/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:183)"} []
[2017-02-15 12:35:25] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\ConnectionException: An exception occured in driver: SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php line 85) {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php:85, Doctrine\\DBAL\\Driver\\PDOException(code: 7): SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 7): SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)"} []
[2017-02-15 12:35:25] php.CRITICAL: Uncaught Exception: An exception occured in driver: SQLSTATE[08006] [7] fe_sendauth: no password supplied {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php:85, Doctrine\\DBAL\\Driver\\PDOException(code: 7): SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 7): SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43, Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The routing file \"/home/brian/projects/tht/app/config/routing.yml\" contains unsupported keys for \"fos_user\": \"resources\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\" in /home/brian/projects/tht/app/config/routing.yml (which is being imported from \"/home/brian/projects/tht/app/config/routing_dev.yml\"). at /home/brian/projects/tht/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:118, InvalidArgumentException(code: 0): The routing file \"/home/brian/projects/tht/app/config/routing.yml\" contains unsupported keys for \"fos_user\": \"resources\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\". at /home/brian/projects/tht/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:183)"} []
[2017-02-15 12:35:25] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\ConnectionException: "An exception occured in driver: SQLSTATE[08006] [7] fe_sendauth: no password supplied" at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php line 85 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php:85, Doctrine\\DBAL\\Driver\\PDOException(code: 7): SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 7): SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43, Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The routing file \"/home/brian/projects/tht/app/config/routing.yml\" contains unsupported keys for \"fos_user\": \"resources\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\" in /home/brian/projects/tht/app/config/routing.yml (which is being imported from \"/home/brian/projects/tht/app/config/routing_dev.yml\"). at /home/brian/projects/tht/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:118, InvalidArgumentException(code: 0): The routing file \"/home/brian/projects/tht/app/config/routing.yml\" contains unsupported keys for \"fos_user\": \"resources\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\". at /home/brian/projects/tht/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:183)"} []
[2017-02-15 12:35:25] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\ConnectionException: An exception occured in driver: SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php line 85) {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php:85, Doctrine\\DBAL\\Driver\\PDOException(code: 7): SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 7): SQLSTATE[08006] [7] fe_sendauth: no password supplied at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)"} []
[2017-02-15 12:38:27] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Config\Exception\FileLoaderLoadException: "The routing file "/home/brian/projects/tht/app/config/routing.yml" contains unsupported keys for "fos_user": "resources". Expected one of: "resource", "type", "prefix", "path", "host", "schemes", "methods", "defaults", "requirements", "options", "condition" in /home/brian/projects/tht/app/config/routing.yml (which is being imported from "/home/brian/projects/tht/app/config/routing_dev.yml")." at /home/brian/projects/tht/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php line 118 {"exception":"[object] (Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The routing file \"/home/brian/projects/tht/app/config/routing.yml\" contains unsupported keys for \"fos_user\": \"resources\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\" in /home/brian/projects/tht/app/config/routing.yml (which is being imported from \"/home/brian/projects/tht/app/config/routing_dev.yml\"). at /home/brian/projects/tht/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:118, InvalidArgumentException(code: 0): The routing file \"/home/brian/projects/tht/app/config/routing.yml\" contains unsupported keys for \"fos_user\": \"resources\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\". at /home/brian/projects/tht/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:183)"} []
[2017-02-15 12:38:32] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Config\Exception\FileLoaderLoadException: "The routing file "/home/brian/projects/tht/app/config/routing.yml" contains unsupported keys for "fos_user": "resources". Expected one of: "resource", "type", "prefix", "path", "host", "schemes", "methods", "defaults", "requirements", "options", "condition" in /home/brian/projects/tht/app/config/routing.yml (which is being imported from "/home/brian/projects/tht/app/config/routing_dev.yml")." at /home/brian/projects/tht/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php line 118 {"exception":"[object] (Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The routing file \"/home/brian/projects/tht/app/config/routing.yml\" contains unsupported keys for \"fos_user\": \"resources\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\" in /home/brian/projects/tht/app/config/routing.yml (which is being imported from \"/home/brian/projects/tht/app/config/routing_dev.yml\"). at /home/brian/projects/tht/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:118, InvalidArgumentException(code: 0): The routing file \"/home/brian/projects/tht/app/config/routing.yml\" contains unsupported keys for \"fos_user\": \"resources\". Expected one of: \"resource\", \"type\", \"prefix\", \"path\", \"host\", \"schemes\", \"methods\", \"defaults\", \"requirements\", \"options\", \"condition\". at /home/brian/projects/tht/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:183)"} []
[2017-02-15 12:38:45] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2017-02-15 12:38:45] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\ConnectionException: "An exception occured in driver: SQLSTATE[08006] [7] could not translate host name "ec2-54-246-108-119.eu-west-1.compute.amazonaws.com" to address: No address associated with hostname" at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php line 85 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[08006] [7] could not translate host name \"ec2-54-246-108-119.eu-west-1.compute.amazonaws.com\" to address: No address associated with hostname at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php:85, Doctrine\\DBAL\\Driver\\PDOException(code: 7): SQLSTATE[08006] [7] could not translate host name \"ec2-54-246-108-119.eu-west-1.compute.amazonaws.com\" to address: No address associated with hostname at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 7): SQLSTATE[08006] [7] could not translate host name \"ec2-54-246-108-119.eu-west-1.compute.amazonaws.com\" to address: No address associated with hostname at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)"} []
[2017-02-15 12:38:46] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"2651a5","_route":"_wdt"},"request_uri":"http://localhost:8000/_wdt/2651a5","method":"GET"} []
[2017-02-15 12:38:47] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-02-15 12:38:50] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2017-02-15 12:38:50] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-02-15 12:38:50] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"f300c8","_route":"_wdt"},"request_uri":"http://localhost:8000/_wdt/f300c8","method":"GET"} []
[2017-02-15 12:38:50] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-02-15 12:38:51] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2017-02-15 12:38:52] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-02-15 12:38:52] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"9aa308","_route":"_wdt"},"request_uri":"http://localhost:8000/_wdt/9aa308","method":"GET"} []
[2017-02-15 12:38:52] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-02-15 12:51:16] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2017-02-15 12:51:16] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\ConnectionException: "An exception occured in driver: SQLSTATE[08006] [7] could not translate host name "ec2-54-246-108-119.eu-west-1.compute.amazonaws.com" to address: Temporary failure in name resolution" at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php line 85 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[08006] [7] could not translate host name \"ec2-54-246-108-119.eu-west-1.compute.amazonaws.com\" to address: Temporary failure in name resolution at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php:85, Doctrine\\DBAL\\Driver\\PDOException(code: 7): SQLSTATE[08006] [7] could not translate host name \"ec2-54-246-108-119.eu-west-1.compute.amazonaws.com\" to address: Temporary failure in name resolution at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 7): SQLSTATE[08006] [7] could not translate host name \"ec2-54-246-108-119.eu-west-1.compute.amazonaws.com\" to address: Temporary failure in name resolution at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)"} []
[2017-02-15 12:51:16] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\ConnectionException: An exception occured in driver: SQLSTATE[08006] [7] could not translate host name "ec2-54-246-108-119.eu-west-1.compute.amazonaws.com" to address: Temporary failure in name resolution at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php line 85) {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[08006] [7] could not translate host name \"ec2-54-246-108-119.eu-west-1.compute.amazonaws.com\" to address: Temporary failure in name resolution at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php:85, Doctrine\\DBAL\\Driver\\PDOException(code: 7): SQLSTATE[08006] [7] could not translate host name \"ec2-54-246-108-119.eu-west-1.compute.amazonaws.com\" to address: Temporary failure in name resolution at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 7): SQLSTATE[08006] [7] could not translate host name \"ec2-54-246-108-119.eu-west-1.compute.amazonaws.com\" to address: Temporary failure in name resolution at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)"} []
[2017-02-15 12:51:16] php.CRITICAL: Uncaught Exception: An exception occured in driver: SQLSTATE[08006] [7] could not translate host name "ec2-54-246-108-119.eu-west-1.compute.amazonaws.com" to address: Temporary failure in name resolution {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[08006] [7] could not translate host name \"ec2-54-246-108-119.eu-west-1.compute.amazonaws.com\" to address: Temporary failure in name resolution at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php:85, Doctrine\\DBAL\\Driver\\PDOException(code: 7): SQLSTATE[08006] [7] could not translate host name \"ec2-54-246-108-119.eu-west-1.compute.amazonaws.com\" to address: Temporary failure in name resolution at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 7): SQLSTATE[08006] [7] could not translate host name \"ec2-54-246-108-119.eu-west-1.compute.amazonaws.com\" to address: Temporary failure in name resolution at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43, Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[08006] [7] could not translate host name \"ec2-54-246-108-119.eu-west-1.compute.amazonaws.com\" to address: Temporary failure in name resolution at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php:85, Doctrine\\DBAL\\Driver\\PDOException(code: 7): SQLSTATE[08006] [7] could not translate host name \"ec2-54-246-108-119.eu-west-1.compute.amazonaws.com\" to address: Temporary failure in name resolution at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 7): SQLSTATE[08006] [7] could not translate host name \"ec2-54-246-108-119.eu-west-1.compute.amazonaws.com\" to address: Temporary failure in name resolution at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)"} []
[2017-02-15 12:51:16] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\ConnectionException: "An exception occured in driver: SQLSTATE[08006] [7] could not translate host name "ec2-54-246-108-119.eu-west-1.compute.amazonaws.com" to address: Temporary failure in name resolution" at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php line 85 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[08006] [7] could not translate host name \"ec2-54-246-108-119.eu-west-1.compute.amazonaws.com\" to address: Temporary failure in name resolution at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php:85, Doctrine\\DBAL\\Driver\\PDOException(code: 7): SQLSTATE[08006] [7] could not translate host name \"ec2-54-246-108-119.eu-west-1.compute.amazonaws.com\" to address: Temporary failure in name resolution at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 7): SQLSTATE[08006] [7] could not translate host name \"ec2-54-246-108-119.eu-west-1.compute.amazonaws.com\" to address: Temporary failure in name resolution at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43, Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[08006] [7] could not translate host name \"ec2-54-246-108-119.eu-west-1.compute.amazonaws.com\" to address: Temporary failure in name resolution at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php:85, Doctrine\\DBAL\\Driver\\PDOException(code: 7): SQLSTATE[08006] [7] could not translate host name \"ec2-54-246-108-119.eu-west-1.compute.amazonaws.com\" to address: Temporary failure in name resolution at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 7): SQLSTATE[08006] [7] could not translate host name \"ec2-54-246-108-119.eu-west-1.compute.amazonaws.com\" to address: Temporary failure in name resolution at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)"} []
[2017-02-15 12:51:16] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\ConnectionException: An exception occured in driver: SQLSTATE[08006] [7] could not translate host name "ec2-54-246-108-119.eu-west-1.compute.amazonaws.com" to address: No address associated with hostname at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php line 85) {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[08006] [7] could not translate host name \"ec2-54-246-108-119.eu-west-1.compute.amazonaws.com\" to address: No address associated with hostname at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php:85, Doctrine\\DBAL\\Driver\\PDOException(code: 7): SQLSTATE[08006] [7] could not translate host name \"ec2-54-246-108-119.eu-west-1.compute.amazonaws.com\" to address: No address associated with hostname at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 7): SQLSTATE[08006] [7] could not translate host name \"ec2-54-246-108-119.eu-west-1.compute.amazonaws.com\" to address: No address associated with hostname at /home/brian/projects/tht/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)"} []
[2017-02-15 12:51:17] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2017-02-15 12:51:17] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-02-15 12:51:17] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"cd3bcc","_route":"_wdt"},"request_uri":"http://localhost:8000/_wdt/cd3bcc","method":"GET"} []
[2017-02-15 12:51:18] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-02-15 12:51:18] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2017-02-15 12:51:19] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-02-15 12:51:19] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"ffaeb9","_route":"_wdt"},"request_uri":"http://localhost:8000/_wdt/ffaeb9","method":"GET"} []
[2017-02-15 12:51:19] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
